File: public/phase-2-20160620022203/C_SHARP/apartment.cs
// Name : Apartment
using System;
namespace Application {
	class apartment {
		public static void Main(string[] args) {
		// input:
		// n :  the size of Shalkan’s apartment plan
		// m : the size of Shalkan’s apartment plan
		// nm :  the description of the apartment plan
		int idx;
		int jdx;
		string[] elements = (Console.ReadLine()).Split(' ');
		int n = Convert.ToInt32(elements[0]);
		int m = Convert.ToInt32(elements[1]);
		char[,] nm = new char[n,m];
		string[] nm_elements = new string[100];
		for(idx=0;idx<n;idx++){
			nm_elements = (Console.ReadLine()).Split(' ');
			for(jdx=0;jdx<m;jdx++){
				nm[idx,jdx] = nm_elements[jdx];
			}
		}


		// write your code here
		// store your results in `new_plan`

		// output
		// Dummy Data
		char[][] new_plan= new char[][] {{'.','*','.','*','.'},{'*','*','*','*','*'},{'.','*','.','*','.'},{'*','*','*','*','*'},{'.','*','.','*','.'}};

for(idx=0;idx<n;idx++){
			Console.WriteLine(string.Join(" ", new_plan[idx]));
		}
		}	
	}
}

File: public/phase-2-20160620022203/C_PLUS_PLUS/apartment.cpp
// Name : Apartment
#include <iostream>
#include <string>
using namespace std;

int main(int argc, char const *argv[]) {
	// input:
	// n :  the size of Shalkan’s apartment plan
	// m : the size of Shalkan’s apartment plan
	// nm :  the description of the apartment plan
	int idx, jdx;
	int n, m;

	cin >> n >> m;
	char nm[2000][2000];

	for(idx=0;idx<n;idx++)
		for(jdx=0;jdx<m;jdx++)
			cin >> nm[idx][jdx];


	// write your code here
	// store your results in `new_plan`

	// output
	// Dummy Data
	char new_plan[2000][2000]={{'.','*','.','*','.'},{'*','*','*','*','*'},{'.','*','.','*','.'},{'*','*','*','*','*'},{'.','*','.','*','.'}};

	for(idx=0;idx<n;idx++) {
		for(jdx=0;jdx<m;jdx++) {
			cout << new_plan[idx][jdx];
		}
		cout << endl;
	}

}